commit 121a78dc5b5933ae188fca449e0db2f616bb908b (refs/changes/46/4746/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-08-27T17:41:55+00:00 (1 year, 1 month ago)
    
    OS-7178 qemu build.sh uses incorrect library paths
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jorge Schrauwen <jorge@blackdot.be>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/build.sh b/build.sh
index 318f51c..553a0ea 100755
--- a/build.sh
+++ b/build.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2012, Joyent Inc., All rights reserved.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 for dir in seabios vgabios kvm/test; do
@@ -37,7 +37,7 @@ echo "==> Running configure"
 KVM_DIR="${KVM_DIR:-$(cd `pwd`/../kvm; pwd)}"
 CC="${CC:-${DESTDIR}/usr/bin/gcc}"
 XCFLAGS="-fno-builtin -I${PNGINC} -isystem ${DESTDIR}/usr/include -msave-args"
-XLDFLAGS="-nodefaultlibs -L${PNGLIB} -L${DESTDIR}/usr/lib -L${DESTDIR}/lib"
+XLDFLAGS="-nodefaultlibs -L${PNGLIB} -L${DESTDIR}/usr/lib/amd64 -L${DESTDIR}/lib/amd64"
 XLDFLAGS="${XLDFLAGS} -Wl,-zfatal-warnings -Wl,-zassert-deflib"
 XLDFLAGS="${XLDFLAGS} -lz -lm -lc -lvnd"
 ./configure \
